src/components/FiltersBar/FiltersBar.tsx

```diff
@@ -0,0 +1,386 @@
+import { add } from 'date-fns'
+import React from 'react'
+import { CSSTransition } from 'react-transition-group'
+
+import knownLicenses from '@/data/knownLicenses.json'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { Button, ButtonProps } from '@/shared/components/Button'
+import { Checkbox } from '@/shared/components/Checkbox'
+import { PopoverDialog } from '@/shared/components/Popover'
+import { RadioButton } from '@/shared/components/RadioButton'
+import { Text } from '@/shared/components/Text'
+import { SvgGlyphClose } from '@/shared/icons'
+
+import {
+  FilterContentContainer,
+  FiltersContainer,
+  FiltersInnerContainer,
+  MobileFilterContainer,
+  OtherFilterStyledIcon,
+  OtherFilterStyledText,
+} from './FiltersBar.styles'
+import { VideoLengthOptions, useFiltersBar } from './useFiltersBar'
+
+import { ActionDialog, ActionDialogProps } from '../ActionDialog'
+import { StyledTitleText } from '../MessageDialog/MessageDialog.style'
+
+export const FiltersBar: React.FC<ReturnType<typeof useFiltersBar>> = ({
+  setVideoWhereInput,
+  videoWhereInput,
+  filters: {
+    setiIsFiltersOpen,
+    isFiltersOpen,
+    dateUploadedFilter,
+    setDateUploadedFilter,
+    matureContentRatingFilter,
+    setMatureContentRatingFilter,
+    paidPromotionalMaterialFilter,
+    setPaidPromotionalMaterialFilter,
+    videoLegnthFilter,
+    setVideoLegnthFilter,
+    licensesFilter,
+    setLicensesFilter,
+  },
+  canClearFilters: {
+    canClearDateUploadedFilter,
+    canClearVideoLegnthFilter,
+    canClearAllFilters,
+    clearAllFilters,
+    clearDateUploadedFilter,
+    clearVideoLegnthFilter,
+    clearLicensesFilter,
+    clearOtherFilters,
+  },
+}) => {
+  const smMatch = useMediaMatch('sm')
+  const betweenBaseAndSMMatch = !smMatch
```

Comment:
Doesn't seem really needed? There's only one condition, it could use `!smMatch` directly right?

Reply:
Kept it for clarity, hard to know what breakpoint is !smMatch